am-ui/webcomponents/create-ad-webcomponent/src/locales/i18n.js

```diff
@@ -17,6 +17,26 @@ export default createI18n({
       upload: 'Upload images',
       uploadThumbnail: 'Upload thumbnail',
       adIsPublished:'Ad is published',
+      errorMessage_title_tooLongOrTooShort: 'Title is too long or too short',
+      errorMessage_address_tooLongOrTooShort: 'Address is too long or too short',
+      errorMessage_region: 'Region missing',
+      errorMessage_expectations_exdeab: 'Please state your expectations',
+      errorMessage_description_exdeab: 'Please provide a description',
+      errorMessage_about_exdeab: 'Please provide a description about yourself',
+      errorMessage_postalCode: 'Postcode missing',
+      errorMessage_email: 'E-Mail wrong',
+      errorMessage_singleFileTooBig: 'One picture is too big',
+      errorMessage_sumOfFilesTooBig: 'The total size of all images is too large',
+      errorMessage_size_NaN: 'Invalid value',
+      errorMessage_floor_NaN: 'Invalid value',
+      errorMessage_price_NaN: 'Invalid value',
+      errorMessage_phone_NaN: 'Invalid phone number',
+      errorMessage_size_missing: 'Living space missing',
+      errorMessage_floor_missing: 'Floors missing',
+      errorMessage_price_missing: 'Rent missing',
```

Comment:
For space, floor, rent is missing the verb "is" -> "Rent is missing"

Reply:
Thanks, changed it